xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/suggest/xwiki.selectize.js

```diff
@@ -135,6 +132,12 @@ define('xwiki-selectize', ['jquery', 'selectize', 'xwiki-events-bridge'], functi
         var wrapper = this.$wrapper;
         wrapper.addClass(this.settings.loadingClass);
       }
+    },
+    onInitialize: function() {
+      this.${escapetool.d}control.on('click', 'a.xwiki-selectize-option-label', function(event) {
```

Comment:
Are you sure this line can be minified? ``${escapetool.d}`` doesn't look like valid JavaScript. This file is minified at build time and then later evaluated for Velocity at runtime so the Velocity code needs to appear as JavaScript code. Assigning ``this.$control`` to a variable before adding the event might be an easier fix.

Reply:
Are you suggesting the following?
```suggestion
      var control = this.${escapetool.d}control;
      control.on('click', 'a.xwiki-selectize-option-label', function(event) {
```

Reply:
Seems like the right way to do it, I'll run some test and apply your suggestion.